configure.py

```diff
@@ -1037,16 +1037,18 @@ def configure_node(o):
   else:
     o['variables']['node_use_dtrace'] = 'false'
 
-  if options.node_use_large_pages and not flavor in ('linux', 'freebsd'):
+  if options.node_use_large_pages and not flavor in ('linux', 'freebsd', 'mac'):
     raise Exception(
-      'Large pages are supported only on Linux Systems.')
-  if options.node_use_large_pages and flavor in ('linux', 'freebsd'):
+      'Large pages are supported only on Linux, FreeBSD and MacOS Systems.')
+  if options.node_use_large_pages and flavor in ('linux', 'freebsd', 'mac'):
     if options.shared or options.enable_static:
       raise Exception(
         'Large pages are supported only while creating node executable.')
     if target_arch!="x64":
       raise Exception(
         'Large pages are supported only x64 platform.')
+    if flavor == 'mac':
+      print("macOS server with 32GB or more is recommended")
```

Comment:
Style: single quotes

Substance: `info('macOS server...')` might be better although plain print() isn't exactly wrong either, it's used in plenty of places.

Reply:
Putting @bnoordhuis's comment above as a suggestion for ease of adoption:

```suggestion
      info('macOS server with 32GB or more is recommended')
```